zebra-consensus/Cargo.toml

```diff
@@ -8,3 +8,17 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+zebra-chain = { path = "../zebra-chain" }
+zebra-state = { path = "../zebra-state" }
+futures-util = "0.3.5"
+tower = "0.3.1"
+
+[dev-dependencies]
+zebra-test-vectors = { path = "../zebra-test-vectors/" }
+color-eyre = "0.3.4"
+eyre = "0.4.2"
+spandoc = { git = "https://github.com/yaahc/spandoc.git" }
+tokio = { version = "0.2.21", features = ["full"] }
```

Comment:
I think this _might_ have negative interactions with the features that are enabled when not using dev-dependencies. AFAIK this bug in cargo was fixed in nightly recently so its probably fine but we might want to open an issue to verify that we're not pulling in more of tokio than wanted when compiling just `zebra-concensus` without tests

Reply:
I used GitHub's "reference in new issue" to open #468.